docs/manifests/network-policies/shippingservice.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: shippingservice
```

Comment:
**Praise:**
I like that way we're naming the NetworkPolicies.
It's simple.
Even the Service Kubernetes objects just re-use the names of the pods (e.g., the `frontend`'s Service is [named `frontend`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/release/v0.3.6/kubernetes-manifests/frontend.yaml#L92)).